browser/net/brave_ad_block_tp_network_delegate_helper.cc

```diff
@@ -33,6 +33,7 @@
 #include "content/public/common/url_constants.h"
 #include "extensions/common/url_pattern.h"
 #include "mojo/public/cpp/bindings/remote.h"
+#include "services/network/host_resolver.h"
```

Comment:
@bridiver `gn_check` complains about this include, and I'm not sure how to get around it.

Reply:
```patch
diff --git a/browser/DEPS b/browser/DEPS
index d97b76b145..9e3ef76e90 100644
--- a/browser/DEPS
+++ b/browser/DEPS
@@ -48,6 +48,7 @@ include_rules = [
   "+services/image_annotation/public",
   "+services/media_session/public",
   "+services/metrics/public",
+  "+services/network/host_resolver.h",
   "+services/network/network_context.h",
   "+services/network/network_service.h",
   "+services/network/public",
```

Reply:
that does it, thanks @darkdh!